lib/js/components/DatePickers/DatePicker/DatePicker.composables.ts

```diff
@@ -0,0 +1,87 @@
+import { onMounted, onUnmounted, Ref, ref, watch } from 'vue';
+import { Instance as DatePickerInstance } from 'flatpickr/dist/types/instance';
+import flatpickr from 'flatpickr';
```

Comment:
I have no idea how to lazy load it

Reply:
@rogatty what you mean about loading and error state?

Reply:
@pioziol123 add delay inside `createDatePicker()` and see if the UX is acceptable or is everything hanging. Maybe we need a loading spinner somewhere?

Then block the flatpickr chunk from being loaded and see if the UX is acceptable. Maybe we need an alert or a dedicated state?